examples/games/rock_paper_scissors/play_rock_paper_scissors.qnt

```diff
@@ -0,0 +1,64 @@
+module rock_paper_scissors {
```

Comment:
Let's enable syntax highlighting by tricking GitHub into thinking this is Bluespec code, since we don't have proper highlighting for Quint on GitHub yet:

```suggestion
// -*- mode: Bluespec; -*-

module rock_paper_scissors {
```